atomics/T1562.008/T1562.008.yaml

```diff
@@ -39,3 +39,42 @@ atomic_tests:
        aws s3 rb s3://#{s3_bucket_name} --force 
     name: sh
     elevation_required: false
+- name: o365-exchange-audit-log-disabled
```

Comment:
Can we update this name  to `Office 365 - Exchange Audit Log Disabled` and add some more details to the description of this atomic. 

ideally, add relevant links in the description so that users can understand these TTPs better